App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { StyleSheet, Text, View, Button } from 'react-native'
+
+export default function App() {
+    return (
+        <View style={styles.container}>
+            <Text style={styles.title}>GitHub Project Boards</Text>
+            <Button
```

Comment:
This button should have some kind of background color. Could you make it red?

Reply:
Changes have been made